src/client/dfs/dfs.c

```diff
@@ -142,8 +142,12 @@ struct dfs {
 	int			amode;
 	/** Open pool handle of the DFS mount */
 	daos_handle_t		poh;
+	/** refcount on pool handle that through the DFS API */
```

Comment:
Is a word missing?
```suggestion
	/** refcount on pool handle that [?] through the DFS API */
```

Reply:
i can update that if a refresh is needed, otherwise since it's not in the header ill update in a different PR.